x-pack/plugins/security_solution/public/overview/components/detection_response/rule_alerts_table/rule_alerts_table.tsx

```diff
@@ -0,0 +1,169 @@
+          <EuiLink
+            data-test-subj="severityRuleAlertsTable-name"
+            href={url}
+            onClick={(ev?: React.MouseEvent) => {
+              if (ev) {
+                ev.preventDefault();
+              }
+              navigateTo({ url });
+            }}
+          >
+            {name}
+          </EuiLink>
+        </EuiToolTip>
+      );
+    },
+  },
+  {
+    field: 'last_alert_at',
+    name: i18n.RULE_ALERTS_COLUMN_LAST_ALERT,
+    render: (lastAlertAt: string) => (
+      <FormattedRelative
+        data-test-subj="severityRuleAlertsTable-lastAlertAt"
+        value={new Date(lastAlertAt)}
+      />
+    ),
+  },
+  {
+    field: 'alert_count',
+    name: i18n.RULE_ALERTS_COLUMN_ALERT_COUNT,
+    render: (alertCount: number, { id }) => {
+      const queryParameter = encodeRisonUrlState({
+        language: 'kuery',
+        query: `kibana.alert.rule.uuid: ${id}`,
+      });
+      const url = getAppUrl({
+        deepLinkId: SecurityPageName.alerts,
+        path: `?query=${queryParameter}`,
+      });
+      return (
+        <EuiToolTip data-test-subj={`${id}-tooltip`} content={i18n.OPEN_RULE_ALERTS_TOOLTIP}>
+          <EuiLink
+            data-test-subj="severityRuleAlertsTable-name"
+            // Doing a hard redirect using href only, due to this bug https://github.com/elastic/kibana/issues/123838
+            // TODO: Uncomment the onClick function and change the parameter to "?filters=" when the bug is fixed.
+            href={url}
+            // onClick={(ev?: React.MouseEvent) => {
+            //   if (ev) {
+            //     ev.preventDefault();
+            //   }
+            //   navigateTo({ url });
+            // }}
+          >
+            {alertCount}
+          </EuiLink>
+        </EuiToolTip>
+      );
+    },
+  },
+  {
+    field: 'severity',
+    name: i18n.RULE_ALERTS_COLUMN_SEVERITY,
+    render: (severity: Severity) => (
+      <EuiHealth color={SEVERITY_COLOR[severity]}>{capitalize(severity)}</EuiHealth>
+    ),
+  },
+];
+
+export const RuleAlertsTable = React.memo<RuleAlertsTableProps>(({ signalIndexName }) => {
+  const { getAppUrl, navigateTo } = useNavigation();
+  const { toggleStatus, setToggleStatus } = useQueryToggle(DETECTION_RESPONSE_RULE_ALERTS_QUERY_ID);
+  const { items, isLoading } = useRuleAlertsItems({
+    signalIndexName,
+    queryId: DETECTION_RESPONSE_RULE_ALERTS_QUERY_ID,
+    skip: !toggleStatus,
+  });
+
+  const columns = useMemo(
+    () => getTableColumns({ getAppUrl, navigateTo }),
+    [getAppUrl, navigateTo]
+  );
+
+  return (
+    <EuiPanel hasBorder data-test-subj="ruleAlertsTablePanel">
+      <HeaderSection
+        id={DETECTION_RESPONSE_RULE_ALERTS_QUERY_ID}
+        title={i18n.RULE_ALERTS_SECTION_TITLE}
+        titleSize="s"
+        toggleStatus={toggleStatus}
+        toggleQuery={setToggleStatus}
+        hideSubtitle
+      />
+      {toggleStatus &&
+        (isLoading || items.length > 0 ? (
```

Comment:
I might be wrong here, but did you mean to have a closing parentheses like this: 
```
(isLoading || items.length > 0 `)` ? (
```
Instead of at the very end? I think it evaluates as expected, but my linter tried to smash the `toggleStatus` with the `isLoading`.  `(toggleStatus && isLoading) || items.length >0 `  It could have just been bad formatting on my part or not having that first open parenthesis there yet though.